src/components/Accordion/contexts/AccordionContext.ts

```diff
@@ -0,0 +1,8 @@
+import { createContext } from 'react';
+
+const AccordionContext = createContext({
+  isActive: (id: number | string | null): boolean => false,
+  onClick: (id?: number | string | null): any => () => {},
```

Comment:
```suggestion
  onClick: (id?: number | string | null): void => () => {},
```

Normally the callbacks do not return a value, so you can just add them as void.